plugin/backend.go

```diff
@@ -109,7 +109,6 @@ func (pm *Manager) Push(name string, metaHeader http.Header, authConfig *types.A
 	if err != nil {
 		return err
 	}
-	defer config.Close()
```

Comment:
Where is this getting closed now? The next statement could cause an early error exit.